konfera/models/ticket.py

```diff
@@ -33,19 +33,11 @@ class Ticket(KonferaModel):
     order = models.ForeignKey('Order', blank=True)
 
     def __str__(self):
-        if self.title:
-            full_name = '{title} {first_name} {last_name}'.format(
-                title=dict(Speaker.TITLE_CHOICES)[self.title],
+        return '{title} {first_name} {last_name}'.format(
+                title=dict(Speaker.TITLE_CHOICES).get(self.title, ''),
                 first_name=self.first_name,
                 last_name=self.last_name
             )
```

Comment:
we need to `.strip()` it because if there is not title the text will start with a space